mod.d.ts

```diff
@@ -1,119 +1,117 @@
-  export const KEY_5: number;
-  export const KEY_6: number;
-  export const KEY_7: number;
-  export const KEY_8: number;
-  export const KEY_9: number;
-  export const KEY_SEMICOLON: number;
-  export const KEY_EQUALS: number;
-  export const KEY_A: number;
-  export const KEY_B: number;
-  export const KEY_C: number;
-  export const KEY_D: number;
-  export const KEY_E: number;
-  export const KEY_F: number;
-  export const KEY_G: number;
-  export const KEY_H: number;
-  export const KEY_I: number;
-  export const KEY_J: number;
-  export const KEY_K: number;
-  export const KEY_L: number;
-  export const KEY_M: number;
-  export const KEY_N: number;
-  export const KEY_O: number;
-  export const KEY_P: number;
-  export const KEY_Q: number;
-  export const KEY_R: number;
-  export const KEY_S: number;
-  export const KEY_T: number;
-  export const KEY_U: number;
-  export const KEY_V: number;
-  export const KEY_W: number;
-  export const KEY_X: number;
-  export const KEY_Y: number;
-  export const KEY_Z: number;
-  export const KEY_LEFT_CMD: number;
-  export const KEY_RIGHT_CMD: number;
-  export const KEY_CONTEXT_MENU: number;
-  export const KEY_NUMPAD0: number;
-  export const KEY_NUMPAD1: number;
-  export const KEY_NUMPAD2: number;
-  export const KEY_NUMPAD3: number;
-  export const KEY_NUMPAD4: number;
-  export const KEY_NUMPAD5: number;
-  export const KEY_NUMPAD6: number;
-  export const KEY_NUMPAD7: number;
-  export const KEY_NUMPAD8: number;
-  export const KEY_NUMPAD9: number;
-  export const KEY_MULTIPLY: number;
-  export const KEY_ADD: number;
-  export const KEY_SEPARATOR: number;
-  export const KEY_SUBTRACT: number;
-  export const KEY_DECIMAL: number;
-  export const KEY_DIVIDE: number;
-  export const KEY_F1: number;
-  export const KEY_F2: number;
-  export const KEY_F3: number;
-  export const KEY_F4: number;
-  export const KEY_F5: number;
-  export const KEY_F6: number;
-  export const KEY_F7: number;
-  export const KEY_F8: number;
-  export const KEY_F9: number;
-  export const KEY_F10: number;
-  export const KEY_F11: number;
-  export const KEY_F12: number;
-  export const KEY_F13: number;
-  export const KEY_F14: number;
-  export const KEY_F15: number;
-  export const KEY_F16: number;
-  export const KEY_F17: number;
-  export const KEY_F18: number;
-  export const KEY_F19: number;
-  export const KEY_F20: number;
-  export const KEY_F21: number;
-  export const KEY_F22: number;
-  export const KEY_F23: number;
-  export const KEY_F24: number;
-  export const KEY_NUM_LOCK: number;
-  export const KEY_SCROLL_LOCK: number;
-  export const KEY_COMMA: number;
-  export const KEY_PERIOD: number;
-  export const KEY_SLASH: number;
-  export const KEY_BACK_QUOTE: number;
-  export const KEY_OPEN_BRACKET: number;
-  export const KEY_BACK_SLASH: number;
-  export const KEY_CLOSE_BRACKET: number;
-  export const KEY_QUOTE: number;
-  export const KEY_META: number;
-}
+export declare const KEY_CANCEL = 3;

```

Comment:
I generated this while doing it myself but chose to not commit it. I think the TypeScript compiler is able to find the declaration without the module name with the help of `types` key from package.json?

Reply:
You can use `yarn pack` and import the tar zip on a separate repository to figure it out. I think it will work for both JS and TypeScript.

Reply:
Yes. I though it'd be safer to just clean build d.ts everytime we build ts; and I tested it in a local ts project, this works.